src/Scope.ts

```diff
@@ -150,6 +160,23 @@ export class Scope {
         return ancestors;
     }
 
+    public hasCachedSymbolForToken(token: Token) {
```

Comment:
I'd rather not have the symbolCache public... So these functions are the public interface to it.

Reply:
Yeah, let's just make it public and scrap these wrappers since they essentially expose all of the reasons to make it private in the first place (get/set/has).